src/renderer/atlas/AtlasEngine.api.cpp

```diff
@@ -619,10 +619,15 @@ void AtlasEngine::_resolveFontMetrics(const wchar_t* requestedFaceName, const Fo
     const auto cellHeight = gsl::narrow<u16>(std::ceil(baseline + descentInPx + halfGapInPx));
 
     {
-        COORD resultingCellSize;
-        resultingCellSize.X = gsl::narrow<SHORT>(cellWidth);
-        resultingCellSize.Y = gsl::narrow<SHORT>(cellHeight);
-        fontInfo.SetFromEngine(requestedFaceName, requestedFamily, requestedWeight, false, resultingCellSize, requestedSize);
+        til::size coordSize;
+        coordSize.X = cellWidth;
+        coordSize.Y = cellHeight;
+
+        til::size coordSizeUnscaled;
+        coordSizeUnscaled.X = coordSize.X * USER_DEFAULT_SCREEN_DPI / _api.dpi;
+        coordSizeUnscaled.Y = coordSize.Y * USER_DEFAULT_SCREEN_DPI / _api.dpi;

```

Comment:
Was this an unrelated bug fix?

Reply:
Oh yes, this one has sneaked in. I noticed this bug when I was running our unit tests and still had AtlasEngine enabled. It reports incorrect viewport sizes, which causes our win32 console APIs to return incorrect values.